src/YUILoader.cc

```diff
@@ -156,6 +157,35 @@ void YUILoader::loadUI( bool withThreads )
     }
 }
 
+void YUILoader::loadTestFrameworkPlugin( )
+{
+        // Do not try to login if test framework is not enabled
+        if( !getenv( YUITest_ENABLE )  && strcmp(getenv( YUITest_ENABLE ), "1") != 0 ) return;
+        yuiMilestone () << "Requested to start http server to control UI." << std::endl;
+
+        if( pluginExists( YUIPlugin_Test ) ) {
+                YUIPlugin uiPlugin( YUIPlugin_Test );
+                if ( uiPlugin.success() )
+                {
+                        yuiMilestone () << "Loading http server to control UI." << std::endl;
```

Comment:
NP: Coding style. Indentation is 4 spaces, not one tab. Each sequence of 8 spaces can be replaced by a tab, but tab width is defined to be 8 in all of the UI code.

Reply:
Ah, I see, that what I was confused with, thanks for the explanation. I'm the one who prefers spaces over tabs. Will fix that.